source/index.rst

```diff
@@ -6,10 +6,13 @@
 Mixxx |version| User Manual
 ***************************
 
-Mixxx is Free [#f1]_ DJ software for Windows, Mac OS X, and Linux. **Mixxx gives
-you everything you need to become a professional DJ** (except talent and hard
-work). Using Mixxx, you can rock the party with your vinyl turntables, your
-:term:`MIDI`/:term:`HID` controllers, or even just your keyboard.
+Mixxx is free [#f1]_ DJ software for Windows, Mac OS X and Linux. It's
+easy to use, so you will start mixing your favorite songs in no time,
+even if you've never DJed before. Mixxx is also
+powerful, so experienced DJs will find all the advanced features they
+would expect. You can perform with Mixxx using :term:`MIDI` and :term:`HID`
+controllers, CD players, vinyl turntables or even just your
+computer's keyboard and mouse.
```

Comment:
> It's easy to use so you can start mixing your favorite songs in no time

I'd keep the "even if you've never DJed before".


> Mixxx offers a variety of features like effects , master-sync, vinyl-scratching and many more for experienced DJs and beginners alike.

I like to enumerate and include links to specific features. However I would ditch the "beginners alike", we already "catched" them in the preceding sentence. In essence what we want to say here is that Mixxx is not a toy, thus "catching" the advanced users. How about:

> Mixxx offers all the features an experienced DJ would expect like effects, Vinyl control, Beatloop rolls, automatic Key detection and many more.

We don't have to enumerate things like master sync here, there's the "New in 1.12" section below.